tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -73,13 +76,15 @@ spec:
     image: public.ecr.aws/kit/clusterloader2:0213bea
     onError: continue
     script: |
+      #!/bin/bash
       ENDPOINT_FLAG=""
       if [ -n "$(params.endpoint)" ]; then
         ENDPOINT_FLAG="--endpoint $(params.endpoint)"
       fi
       aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region)
       cat $(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml
       ENABLE_EXEC_SERVICE=false /clusterloader --kubeconfig=/root/.kube/config --testconfig=$(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml --testoverrides=$(workspaces.source.path)/overrides.yaml --nodes=$(params.nodes) --provider=eks --report-dir=$(workspaces.results.path) --alsologtostderr --v=2
+      echo $? | tee $(results.cl2-result.path) 
     timeout: 30000s
   - name: upload-results
     image: alpine/k8s:1.22.6
```

Comment:
@hakuna-matatah Do we have any best practice about which image to use between `alpine/k8s` and `amazon/aws-cli`? It seems `alpine/k8s` supports awscli v1 while `amazon/aws-cli` support awscli v2. Should we use awscli v2 here?